src/server/editionsPage.tsx

```diff
@@ -34,37 +37,47 @@ const docParser = JSDOM.fragment.bind(null);
 // ----- Functions ----- //
 
 const styles = `
-    ${pageFonts}
+	${pageFonts}
+
 	html {
-		height: 100%;
-	}
-    body {
 		margin: 0;
 		height: 100%;
 		min-height: 100%;
-    }
+	}
 `;
 
-const renderHead = (
+function renderHead(
+	item: Item,
 	request: RenderingRequest,
 	thirdPartyEmbeds: ThirdPartyEmbeds,
 	itemStyles: string,
 	emotionIds: string[],
-): string => {
-	const cspString = cspEditions(
-		{ scripts: [], styles: [styles, itemStyles] },
+	inlineStyles: boolean,
+): string {
+	const generalStyles = styles;
+	const isEditions = true;
+	const cspString = csp(
+		item,
+		{
+			scripts: [atomScript],
+			styles: [generalStyles, itemStyles, atomCss],
+		},
 		thirdPartyEmbeds,
+		inlineStyles,
+		isEditions,
 	);
+	const meta = h(Meta, { title: request.content.webTitle, cspString });
 
 	return `
-		<meta charSet="utf-8" />
-		<title>${request.content.webTitle}</title>
-		<meta name="viewport" content="initial-scale=1" />
-		<meta http-equiv="Content-Security-Policy" content="${cspString}" />
-        <style>${styles}</style>
+        ${renderToString(meta)}
+        <link rel="stylesheet" type="text/css" href="/fontSize.css">
+        <style>${generalStyles}</style>
         <style data-emotion-css="${emotionIds.join(' ')}">${itemStyles}</style>
+        <script id="targeting-params" type="application/json">
+            ${JSON.stringify(request.targetingParams)}
+        </script>
```

Comment:
Is Editions starting to use the targeting params?

Reply:
No don't think so, good call.